From: John Reese <jreese@leetcode.net>
Date: Tue, 25 Nov 2008 19:54:32 +0000 (-0500)
Subject: Fix #9890: improper comparison of file extensions in file_download.php
X-Git-Url: http://git.mantisbt.org/?p=mantisbt.git;a=commitdiff_plain;h=99903fda7b565afc30a9ae9e37f9cae2053430ff

Fix #9890: improper comparison of file extensions in file_download.php
---

diff --git a/file_download.php b/file_download.php
index 067b03a..8074cc4 100644
--- a/file_download.php
+++ b/file_download.php
@@ -91,7 +91,7 @@
 	header( 'Content-Length: ' . $v_filesize );
 	$t_filename = file_get_display_name( $v_filename );
 	$t_inline_files = explode(',', config_get('inline_file_exts', 'gif'));
-	if ( in_array( file_get_extension($t_filename), $t_inline_files ) ) {
+	if ( in_array( strtolower( file_get_extension($t_filename) ), $t_inline_files ) ) {
 		$t_disposition = ''; //'inline;';
 	} else {
 		$t_disposition = ' attachment;';
